### PR TITLE
1598: WIP Update RHEL7 install build and install instructions.

### PR DESCRIPTION
**Pull request by tst-jlarocco**

### CODE REVIEW

File: earth_enterprise/BUILD_RHEL_CentOS.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ sudo yum install -y epel-release
 
 ```bash
 sudo yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+sudo yum-config-manager --enable rhel-7-server-rhui-optional-rpms
```
Comment (tst-lsavoie) [inline earth_enterprise/BUILD_RHEL_CentOS.md]: I think we need different sections here that indicate what the commands are on regular machines vs on AWS machines.

File: earth_enterprise/BUILD_RPMS.md [hunk 2]
```diff
@@ -87,6 +87,8 @@ sudo yum install -y epel-release
 
 ```bash
 sudo yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+sudo yum-config-manager --enable rhel-7-server-rhui-optional-rpms
```
Comment (tst-lsavoie) [inline earth_enterprise/BUILD_RPMS.md]: Same: should show the commands for AWS and non-AWS machines.

File: earth_enterprise/BUILD_RHEL_CentOS.md (hunk 1 above)
Comment (tst-jlarocco) [reply earth_enterprise/BUILD_RHEL_CentOS.md]: Okay, I didn't realize there were different versions for AWS and non-AWS.
Is non-AWS the same as what was there before?  I'll see what I can find out.

Comment (tst-lsavoie) [reply earth_enterprise/BUILD_RHEL_CentOS.md]: As far as I know, this is what's needed for non-AWS machines:
```
 sudo subscription-manager repos --enable=rhel-7-server-optional-rpms
 sudo subscription-manager repos --enable=rhel-7-server-extras-rpms
```
but I haven't tested that recently.

**Comment by tst-jlarocco** [comment]
New PR with additional changes and targeting master branch:
https://github.com/google/earthenterprise/pull/1630

**Pull request closed without merging**